topics/maven-assembly-plugin.md

```diff
@@ -0,0 +1,54 @@
+[//]: # (title: Maven Assembly plugin)
+
+<microformat>
+<p>
+<control>Initial project</control>: <a href="https://github.com/ktorio/ktor-maven-sample/">ktor-maven-sample</a>
```

Comment:
Why don't we have such a thing for the Gradle Application plugin﻿ topic?

Reply:
because for Gradle we take a ready project and don't change anything there